book/layout.md

```diff
@@ -354,10 +372,10 @@ document always starts in the same place it's pretty simple:
 ``` {.python}
 class DocumentLayout:
     def layout(self):
-        self.w = WIDTH
         # ...
-        child.x = self.x = 0
-        child.y = self.y = 0
+        self.w = WIDTH - 2*HSTEP
```

Comment:
Why the change here?

Reply:
This is unrelated to the others, actually, but without padding there seems to be some kind of border that Tk adds by default which obscures some letters, and students wondered if that was a bug in their code.

Reply:
The padding comes back next chapter anyway.

Reply:
OK can you add a comment explaining what's going on?